From foo@baz Fri Apr  6 10:27:52 CEST 2018
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 6 Apr 2018 09:08:12 +0200
Subject: Revert "spi: bcm-qspi: shut up warning about cfi header inclusion"

From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

This reverts commit c30e6636ce101fd61331092c490b9d9c55b2d143.

Florian writes:
	Sorry for noticing so late, but this appears to be bogus, there
	is no MTD_NORFLASH symbol being defined in 4.9, in fact I can't
	find this Kconfig symbol in any kernel version, so this
	effectively results in the driver no longer being selectable, so
	this sure does silence the warning.

It's not good to just disable a whole driver :(

So let's revert the patch for now, Arnd can work on a better build
fix...

Reported-by: Florian Fainelli <f.fainelli@gmail.com>
Cc: Arnd Bergmann <arnd@arndb.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/spi/Kconfig |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/spi/Kconfig
+++ b/drivers/spi/Kconfig
@@ -156,7 +156,6 @@ config SPI_BCM63XX_HSSPI
 config SPI_BCM_QSPI
 	tristate "Broadcom BSPI and MSPI controller support"
 	depends on ARCH_BRCMSTB || ARCH_BCM || ARCH_BCM_IPROC || COMPILE_TEST
-	depends on MTD_NORFLASH
 	default ARCH_BCM_IPROC
 	help
 	  Enables support for the Broadcom SPI flash and MSPI controller.
